pkgs/leak_tracker_testing/lib/src/leak_testing.dart

```diff
@@ -34,7 +34,7 @@ import 'matchers.dart';
 @immutable
 class LeakTesting {
   const LeakTesting._({
-    this.ignore = true,
+    this.ignore = false,
```

Comment:
if we are now using LeakTracking.enabled to set whether leak tracking is enabled, do we even need this `ignore` field anymore?

Reply:
Yes, for some tests we want to turn off leak tracking. 
LeakTesting.enabled cannot be turned off because it indicates that leak tracking infrastructure is used for this process. 

ignore means to ignore just for this test.